### PR TITLE
Add countdown to start game

### PR DESCRIPTION
**Pull request by teinarss**
Needs some polish, but would be nice to have some general feedback, especial on `TickAction` could probably merge it with  `ActionQueue` but with a performance cost(?),

### CODE REVIEW

File: OpenRA.Game/Network/UnitOrders.cs [hunk 1]
```diff
@@ -141,7 +141,10 @@ internal static void ProcessOrder(OrderManager orderManager, World world, int cl
 						else
 							Game.AddSystemLine(ServerChatName, "The game has started.");
 
-						Game.StartGame(orderManager.LobbyInfo.GlobalSettings.Map, WorldType.Regular);
+						var gameStarter = new GameStarter(orderManager.LobbyInfo.GlobalSettings.Map);
+
```
Comment (abcdefg30) [inline OpenRA.Game/Network/UnitOrders.cs]: Style nit: Please remove this empty line.

File: OpenRA.Game/Game.cs [hunk 2]
```diff
@@ -940,4 +948,32 @@ public static void FinishBenchmark()
...
+		public void Add(ITickAction action)
+		{
+			actions.Add(action);
+		}
+
+		public void Tick()
+		{
+			var toRemove = new List<ITickAction>();
+			foreach (var action in actions)
+			{
```
Comment (abcdefg30) [inline OpenRA.Game/Game.cs]: Style nit: Remove the braces here please.

**Review by abcdefg30 — COMMENTED** [review]
As said on IRC, :-1: from me for adding a timer. I think it unnecessarily wastes player time.

**Comment by GraionDilach** [comment]
Could this be controlled from mod.yaml? Generals had thsi feature.

**Comment by teinarss** [comment]
closing for now, need to get back to the drawing board.

**Pull request closed without merging**